### DISCUSSION TITLE
SunbirdEd-portal repo description pointing to dead link of installation

### DISCUSSION THREAD

**Discussion by baljeetsingh** [Sunbird-Ed/Community#570] [Documentation] [answered]
![image](https://github.com/Sunbird-Ed/Community/assets/344965/c8001e30-a0cc-4cec-becf-6d4045cf8e50)

https://github.com/Sunbird-Ed/SunbirdEd-portal/tree/release-5.1.0
Repo About description is pointing to wrong url of installation docs
Currently it reads as 
```
About
Web Portal for sunbird software. Provides the web interfaces for all functionality of Sunbird. 

Find the installation instructions at: 
http://www.sunbird.org/developer-docs/installation/
```

correct url is
`https://ed.sunbird.org/use/installing-ed-platform`

I feel that over all message could be changed to this (include bb home), so that user / dev can get more details about it

```
Web Portal for sunbird software. Provides the web interfaces for all functionality of Sunbird. 

Know more about sunbird-ed
https://ed.sunbird.org/learn/readme

Find the installation instructions at: 
https://ed.sunbird.org/use/installing-ed-platform
```

**Comment by Krishnaj20** [comment]
@rajeevsathish @chitranshu-keshav @vinukumar-vs - Please look into this query and respond.

**Comment by vinukumar-vs** [comment, marked as answer]
@baljeetsingh Thanks for highlighting the issue. We will update it ASAP.

@rajeevsathish Can you look into it & update the description to point to microsite installation document.
